DOCUMENTATION.md

```diff
@@ -1251,6 +1290,22 @@ Get an application config.json.
 
 #### Returns:
     dict: application config.json
+### Function: get_device_os_semver_with_variant(os_version, os_variant)
```

Comment:
@nghiant2710 can we have an example for this function call as well?

Reply:
@imrehg, done, can you please check this PR again?

Reply:
That part is good, thanks!